docs/source/user-guide/vision/supported_tasks_and_formats.rst

```diff
@@ -0,0 +1,124 @@
+.. _supported_tasks_and_formats:
+
+===========================
+Supported Tasks and Formats
+===========================
+
+In order to accurately analyze the images, labels and predictions provided in the
+:doc:`VisionData object </user-guide/vision/vision_data_class>`,
+they must be provided in deepchecks accepted format. In this section we will describe the supported formats
+for each supported task type.
+
+The data formats are validated at the creation of the VisionData object based on the first batch of data. However,
+in addition to the automatic validations, it is also important to make sure visually that the data was provided in the
+correct format via the :func:`head <deepchecks.vision.VisionData.head>` functionality.
+
+
+Common Formats
+==============
+All types of data are loaded in a batch by batch manner meaning that each data input contain
+information about several samples. In order to support this, all data inputs are expected to be provided
+as either an iterable in which each entry is the data for a single sample or as a high dimension array or tensor in
+which the first dimension is the number of samples.
+
+Image Format
+------------
+Each image in the batch can either be a `PIL image object <https://pillow.readthedocs.io/en/stable/reference/Image.html>`_
+or a ``[H, W, C]`` 3D array (can be either numpy or a tensor).
+For the array format, the first dimension must be the image y axis, the second being the image x axis, and the
+third being the number of channels.
+The numbers in the array should be integers in the range [0, 255]. Color images should be in RGB format and
+have 3 channels, while grayscale images should have 1 channel.
+
+
+Image Identifiers Format
+------------------------
+Image identifiers is a mechanism that allow identifying relevant images in the check results so that they can
+be easily retrieved from the original dataset by the user. Common uses for this can be identifying images by
+the path to the image or by their name or id in a database.
+Image identifiers need to be provided as an iterable of strings.
+
```

Comment:
Would add a sentence that says:

"As label and prediction format are dependent on the specific task type, the following sections will explain the accepted format per task type:"

Or something like that.

Reply:
Then maybe before, in the beginning of this section? I feel like it's missing.